free-1.md

```diff
@@ -1,68 +1,170 @@
 # Free project 1
 
+
+
+An RPG mapmaker with mixed tile sizes and shapes. It would let users create or upload tiles of certain sizes, then specify on the map which patterns belong where by clicking and dragging map regions and selecing the tile to fill the area. There would also be an option to designate what a "border" to the tile looks like, and then the area would have the border on the outside. Tiles that don't fill a square could also have transparency and be layered over others, so borders wouldn't have to be sharp and linear. Once the areas and patterns are specified, an image would be produced. "Objects" could also be placed on the scene, layered above tiled areas.
+
+
+
 ## The user and a language
+
 This section describes who the project would serve and why a language might be a
+
 good way to meet their needs.
 
 
+
+The user would be game developers (or perhaps bored artists). Unfortunately, this feels more like an application than a language, but map generation is a very useful tool.
+
+
+
 ### What's the need?
+
 _What need is met by your idea? Who are you helping? What is that person's
+
 experience like now? What would their experience be like if you could help 
+
 them?_
 
 
+
+Well, existing map generators that I'm aware of generally have set tile sizes, which limits the ability to quickly design maps that don't look as blocky or aren't constrained by having every piece of the map fill a tile of the same size. But to have complete control over a map would require illustrating it entirely from scratch, which would take too long. This would be a tool that might keep the process quick while allowing for more artistic design decisions.
+
+
+
 ### Why a language?
+
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
 
+
+If existing map makers count as DSLs, then these tools have already proven to address this type of need.
+
+
+
 ### Why you?
+
 _What excites you about this idea? How did you come up with it?_
 
 
+
+Words cannot express my interest in game design.
+
+Map generation is one of the basics of game dev tools, and free-to-use tools can always be improved on.
+
+
+
 ### Domain
+
 _Describe the project's domain in five words._
 
 
+
+Tile map editing and generation
+
+
+
 ### Interface (syntax)
+
 _How might the user interact with the language? What does programming look 
+
 like? Why is this the right way to interact with the problem domain?_ 
 
 
+
+A "program" would be a breakdown of a map, with tile patterns specified for each area, potentially how the tiling should be aligned, and with any objects to be layered over this.
```

Comment:
I notice that your description of the interface of the language is medium-agnostic. This would be an equally valid description for a drag and drop application or for a text based specification.

Is the ambiguity intentional? Do you think that interface is at a higher level than these details?